layer_metadata_restorer_test.go

```diff
@@ -68,7 +59,7 @@ func testLayerMetadataRestorer(t *testing.T, when spec.G, it spec.S) {
 
 		when("app and cache metadata are not present", func() {
 			it("does not restore any metadata", func() {
-				_, err := layerAnalyzer.Restore(buildpacks, layersMetadata, nil)
+				err := layerAnalyzer.Restore(buildpacks, layersMetadata, cacheMetadata)
```

Comment:
Maybe it's better to rename `cacheMetadata` to `emptyCacheMetadata` for readability?

Reply:
Maybe... but that's a bit outside the scope of this PR.

Reply:
You're right. `cacheMetadata` is sometimes empty but sometimes is not.
I'm ok with leaving it as is.